docs/04-resources/counter.md

```diff
@@ -0,0 +1,70 @@
+---
+title: cloud.Counter 
+id: counter
+description: A built-in resource for representing an container for numbers in the cloud.
+keywords: [Wing reference, Wing language, language, Wing sdk, Wing programming language, Counter]
+---
+
+The `cloud.Counter` resource represents a stateful container for one or more numbers in the cloud.
```

Comment:
Are these md files meant to document target state or current. I think they should be current, so maybe we dont say the `more` part just yet?

Reply:
Yes, good call.
```suggestion
The `cloud.Counter` resource represents a stateful container for a number in the cloud.
```